_includes/manuals/1.11/4.2.6_matchers.md

```diff
@@ -15,7 +15,7 @@ Overrides are processed in the order of precedence set in the *Order* field, fro
 Example attributes that may be listed are:
 
 * `fqdn` - host's FQDN ("host.example.com")
-* `hostgroup` - full name of the host group, including parents ("Europe/Web servers")
+* `hostgroup` - full name of the host group, including parents ("Europe/Web servers"). Matchers on hostgroups can be inherited by their children, see documentation for `host_group_matchers_inheritance` in [configuration options.](http://www.theforeman.org/manuals/1.11/index.html#3.5.2ConfigurationOptions)
```

Comment:
1) "host groups"
2) link using a relative URL and use `{{page.version}}` in the path, see other links in the document